seed/data_importer/meters_parser.py

```diff
@@ -264,16 +306,22 @@ def _parse_meter_readings(self, details, meter_details, start_time, end_time):
             'conversion_factor': conversion_factor
         }
 
-        meter_identifier = '-'.join([str(v) for k, v in meter_details.items()])
```

Comment:
This was affected by upstream changes (property_id to property_ids). Specifically, the code is modified to run through _all_ property_ids, instead of just _one_ property_id.

The changes here are mostly just reordering and indentation.